README.md

```diff
@@ -36,7 +36,7 @@ In our original work (NeurIPS 2023) we used GPT3.5 and GPT4 as part of our agent
 Conda is an easy-to-use Python interface that supports launching [Jupyter Notebooks](https://jupyter.org/). If you are completely new to this, we recommend checking out the [docs here for getting conda installed](https://deeplabcut.github.io/DeepLabCut/docs/beginner-guides/beginners-guide.html#beginner-user-guide). Otherwise, proceed to use one of [our supplied conda files](https://github.com/AdaptiveMotorControlLab/AmadeusGPT/tree/main/conda). As you will see we have minimal dependencies to get started, and [here is a simple step-by-step guide](https://deeplabcut.github.io/DeepLabCut/docs/installation.html#step-2-build-an-env-using-our-conda-file) you can reference for setting it up (or see [BONUS](README.md#bonus---customized-your-conda-env) below). Here is the quick start command:
 
 ```bash
-conda env create -f amadeusGPT.yml
+conda env create -f conda/amadeusGPT.yml
```

Comment:
please drop conda; that is not useful is one downloads the file, for example.

Reply:
That's the PR from Alex though.  Marc also told me he was confused why there was no conda/.

Reply:
well, if one is within the conda/ folder this cannot be run either, so it's best to keep it not there and use that Q as an opportunity to educate :D